myuw/static/js/card/instructor_schedule/load_course_cards.js

```diff
@@ -81,5 +86,10 @@ var InstructorCourseCards = {
             course_id = course_id.replace(/[^a-z0-9]/gi, '_');
             WSData.log_interaction("open_course_canvas_website_"+course_id, term);
         });
+    },
+
+    is_disabled: function () {
+        return (window.disabled_features.hasOwnProperty('instructor_schedule')
```

Comment:
Can you make an api for this?  If it's going to be feature by feature, i wouldn't want to need all of that duplicated per feature.